modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/internal/freemarker/tool/java/parser/DTOOpenAPIParser.java

```diff
@@ -281,4 +269,30 @@ private static String _getPropertyType(
 		return propertyType;
 	}
 
+	private static boolean _isObject(Schema schema, String type) {
+		if (Objects.equals("object", type) &&
+			(schema.getAdditionalPropertySchema() == null)) {
+
+			return true;
+		}
+
+		return false;
+	}
+
+	private static boolean _isSchema(Schema schema) {
+		Items items = schema.getItems();
+
+		if (_isObject(schema, schema.getType()) ||
+			(schema.getReference() != null) ||
+			(schema.getAllOfSchemas() != null) ||
```

Comment:
It's weird but I think there are some allOf that are returning false... although it makes no sense to me... For example parentOrganization in Organization schema of Headless Admin User

Reply:
Ok, understood now 🙂